src/firebase/firebase.ts

```diff
@@ -382,7 +382,7 @@ export async function fetchCoinbasePaySenders() {
 
 async function fetchListFromFirebase(path: string) {
   if (!FIREBASE_ENABLED) {
-    return []
+    return null
```

Comment:
make the return type `EventChannel<NotUndefined> | null` which is more clear than an empty list